packer/install-dependencies.sh

```diff
@@ -12,15 +12,7 @@ cat /proc/version | tee -a ${RELEASE_NOTES_FILEPATH}
 echo ""
 echo "Components downloaded in this VHD build (some of the below components might get deleted during cluster provisioning if they are not needed):" >> ${RELEASE_NOTES_FILEPATH}
 
-if [[ ${UBUNTU_RELEASE} == "18.04" ]]; then
-  overrideNetworkConfig
-fi
-
-ETCD_VERSION="3.2.25"
-ETCD_DOWNLOAD_URL="https://acs-mirror.azureedge.net/github-coreos"
-installEtcd
-echo "  - etcd v${ETCD_VERSION}" >> ${RELEASE_NOTES_FILEPATH}
-
+applyCIS
```

Comment:
some of other CIS stuff needs to happen after we setup the other things in this script so should we call this at the end of the Packer script instead of before everything else?

Reply:
Yeah, that works.